actions/package-client-credentials

```diff
@@ -5,9 +5,9 @@
 source ~/.bash_aliases
 mkdir -p etcd_credentials
 
-cp $ETCDCTL_CERT_FILE etcd_credentials/client.crt
-cp $ETCDCTL_KEY_FILE etcd_credentials/client.key
-cp $ETCDCTL_CA_FILE etcd_credentials/ca.crt
+cp $ETCDCTL_CERT etcd_credentials/client.crt
+cp $ETCDCTL_KEY etcd_credentials/client.key
+cp $ETCDCTL_CACERT etcd_credentials/ca.crt
```

Comment:
Looks like this change is in conflict with https://github.com/charmed-kubernetes/layer-etcd/pull/186. You'll need to resolve the conflict before we can merge this.